README.md

```diff
@@ -2,6 +2,8 @@
 
 The official [MongoDB](https://www.mongodb.com/) driver for Node.js.
 
+**Upgrading to version 5? Take a look at our [upgrade guide here](https://github.com/mongodb/node-mongodb-native/blob/HEAD/etc/notes/CHANGES_5.0.0.md)!**
+
 **Upgrading to version 4? Take a look at our [upgrade guide here](https://github.com/mongodb/node-mongodb-native/blob/HEAD/etc/notes/CHANGES_4.0.0.md)!**
```

Comment:
We can remove this now, and same with the table entry. Or if we want to surface it still perhaps a section more towards the bottom of the readme that also mentions the 3.x version of that doc? (An "upgrading?" section)